pom.xml

```diff
@@ -75,7 +75,7 @@
     <jackson.version>2.17.0</jackson.version>
     <junit.version>5.10.1</junit.version>
     <mockito.version>4.5.0</mockito.version>
-    <flow.version>24.4-SNAPSHOT</flow.version>
+    <flow.version>24.5-SNAPSHOT</flow.version>
```

Comment:
Could we postpone upgrading `Flow` to the near future when it's released? Now, it is breaking the build.

Reply:
Seems to work now.